server/api/v1/screenshots/[id]/index.delete.ts

```diff
@@ -13,5 +13,16 @@ export default defineEventHandler(async (h3) => {
       statusMessage: "Missing screenshot ID",
     });
 
-  return await screenshotManager.delete(screenshotId);
+  const result = await screenshotManager.get(screenshotId);
```

Comment:
I think the auth check should happen in the screenshotManager.get(), and it returns undefined both on not found and not authorized. This avoids leaking information about screenshots that exist but we don't have access to.

Reply:
I was mostly relying on the idea that the image metadata, doesn't matter too much, but I'll move it over later today though.